app/app.go

```diff
@@ -1112,14 +1197,6 @@ func (app *AltheaApp) SimulationManager() *module.SimulationManager {
 // API server.
 func (app *AltheaApp) RegisterAPIRoutes(apiSvr *api.Server, apiConfig config.APIConfig) {
 	clientCtx := apiSvr.ClientCtx
-	// SDK /node_info, /syncing, /blocks, and /validatorsets REST endpoints
-	rpc.RegisterRoutes(clientCtx, apiSvr.Router)
-
-	// Note: Delegates requests to the EVM if given a hash variable with a leading "0x"
```

Comment:
I know we use the evm rpc in the tests, maybe we should add a quick request to the standard rpc just to make sure it's working. Not sure if this code was supposed to go somewhere else or just isn't needed anymore.